test/sync.test.js

```diff
@@ -332,6 +338,8 @@ describe('Sync protocol', function () {
       await log1.append('hello9')
       expectedEntry2 = await log1.append('hello10')
 
+      expectedEntry = expectedEntry2
+
```

Comment:
This seems extraneous as it’s not used in the code below, right?

Reply:
expectedEntry is needed for determining whether syncedEventFired is true, here: 

```
      const onSynced = async (bytes) => {
        syncedHead = await Entry.decode(bytes)
        if (expectedEntry) {
          syncedEventFired = expectedEntry.hash === syncedHead.hash
        }
      }
```

This is probably a bit messy and should be refactored at a later stage.

Reply:
Refactored.